src/components/StringField.tsx

```diff
@@ -41,6 +43,34 @@ function StringField(props: StringFieldProps) {
   let invalid: boolean =
     isAboveMaxLength || isBelowMinLength || isNotMatchingPattern;
 
+  const errorMessages = {
```

Comment:
Do errors have a unique error code? `error.extensions.code` or is that the same for every error?

Reply:
Instead of `captionError?.message.includes("Invalid organization"` we could have the messages as the key in the object ```js
 {
  "Invalid organization": "Invalid organization",
 ...
 }
```

then you can do ```js
const captionErrorMsg = messages[captionError?.message] || errorMessages.internalServerError;
```

Reply:
> Do errors have a unique error code? `error.extensions.code` or is that the same for every error?

Unfortunately a couple of different errors (chiefly `Invalid image URL`, `Invalid image hostname`, and `Invalid organization`) all use the same code so I couldn't differentiate them that way.



> Instead of `captionError?.message.includes("Invalid organization"` we could have the messages as the key in the object

I had initially wanted to do this however the "missing permission" error uses a string literal for it's error message that contains the organisation ID so I couldn't use that as a key as it would be different each time 😅